.github/workflows/pr-gpu.yaml

```diff
@@ -15,7 +15,7 @@ concurrency:
 jobs:
   pytest-gpu:
     name: ${{ matrix.name }}
-    runs-on: ubuntu-latest # todo: switch to linux-ubuntu-latest later
+    runs-on: ubuntu-latest  # todo: switch to linux-ubuntu-latest later
```

Comment:
TODO

Reply:
I just turned all "todo"s  into "TODO"s.

Reply:
why can't you switch to linux-ubuntu-latest?

Reply:
linux-ubuntu-latest is a custom runner that we at Databricks have. We have not fully set it up yet for this repo so we are using ubuntu-latest right now. @mihir-db previously gave the okay to use this runner.